src/projections/Tools/Timeline/EntryMethodObject.java

```diff
@@ -91,9 +93,11 @@ protected EntryMethodObject(Data data,  TimelineEvent tle,
 	
 		this.data = data;
 		beginTime = tle.BeginTime;
-		endTime   = tle.EndTime;
-		cpuBegin  = tle.cpuBegin;
-		cpuEnd    = tle.cpuEnd;
+		elapsedTime = (int)(tle.EndTime - tle.BeginTime);
+		// If the incoming RecvTime is 0, then it is invalid, so use MIN_VALUE to represent it in the offset
+		recvTimeOffset = (tle.RecvTime == 0) ? Integer.MIN_VALUE : (int)(tle.RecvTime - tle.BeginTime);
+		cpuBegin = tle.cpuBegin;
+		cpuElapsed = (int)(tle.cpuEnd - tle.cpuBegin);
```

Comment:
Is it worth throwing an error if the time is greater than what the int allows you to store? 35 minutes seems like it should never happen, but if it does, it might be hard to figure out why projections logs are all screwed up.

Reply:
Fixed in 9295669.